src/args.ts

```diff
@@ -96,4 +96,14 @@ export let args : ArgDescriptor[] = [
     description: 'Path to h2-push manifest',
     type: String,
   },
+  {
+    name: 'proxy-root',
```

Comment:
Maybe name this `proxy-path`, since it's used in the location of the path argument of `app.get()`, etc.